src/windowmanagement/data/MenuItems.tsx

```diff
@@ -113,7 +113,7 @@ export const menuItemsData: Record<string, MenuItem> = {
     ),
     renderIcon: (size) => <CalendarIcon size={size} />,
     preferredPosition: 'float',
-    floatPosition: { offsetY: 100, offsetX: 370, width: 410, height: 520 },
+    floatPosition: { offsetY: 100, offsetX: 370, width: 410, height: 600 },
```

Comment:
Why this massive height increase? Tried it on 1080p monitor and there is a lot of extra space at the bottom now?

Reply:
For when you press the lock to not get the scrollbar. We could be ok with the scrollbar though, thats ok with me.

Reply:
I think that feature is so rarely used i would say ita fine. I'd be more annoyed with the extra dead space att the bottom than the scrollbar appearing once in a blue moon